CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.0.0-beta2 (unreleased)
+
+* [Breaking] Rename `modules/rpc` to `modules/yarpc`
```

Comment:
yes, the yaml key also changes from "rpc" to "yarpc". It is not immediately obvious from this changelog entry that service owners should update their config